Add log setup and configuration (#3672)
DataDog/integrations-core: jboss_wildfly/README.md

2. [Restart the Agent][5].

#### Log Collection

> Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer: ```suggestion
To enable collecting logs in the Datadog Agent, update `logs_enabled` in `datadog.yaml`:
```

```
    logs_enabled: true
```

Next, edit `jboss_wildfly.d/conf.yaml` by uncommenting the `logs` lines at the bottom. Update the logs `path` with the correct path to your JBoss log files.

```yaml
logs:
 - type: file
   path: /opt/jboss/wildfly/standalone/log/*.log
   source: jboss_wildfly

>    service: myapplication

Reviewer 2: ```suggestion
   service: <APPLICATION_NAME>
```
